streams/src/test/java/org/apache/kafka/streams/integration/EosBetaUpgradeIntegrationTest.java

```diff
@@ -113,14 +114,7 @@
     private static final List<KeyValue<KafkaStreams.State, KafkaStreams.State>> CRASH =
         Collections.unmodifiableList(
             Collections.singletonList(
-                KeyValue.pair(KafkaStreams.State.RUNNING, KafkaStreams.State.ERROR)
-            )
-        );
-    private static final List<KeyValue<KafkaStreams.State, KafkaStreams.State>> CLOSE_CRASHED =
-        Collections.unmodifiableList(
-            Arrays.asList(
-                KeyValue.pair(KafkaStreams.State.ERROR, KafkaStreams.State.PENDING_SHUTDOWN),
-                KeyValue.pair(KafkaStreams.State.PENDING_SHUTDOWN, KafkaStreams.State.NOT_RUNNING)
+                KeyValue.pair(KafkaStreams.State.RUNNING, State.PENDING_ERROR)
```

Comment:
Should we extend it to have two state transitions, `RUNNING -> PENDING_ERROR` and `PENDING_ERROR -> ERROR` ?

Reply:
Actually it could probably be just `PENDING_ERROR -> ERROR` because it could be in `RUNNING` or `REBALNCING` previously